docs/support.rst

```diff
@@ -0,0 +1,39 @@
+Support
+=======
+
+Python Versions
+---------------
+
+artman is currently tested with Python 2.7, Python 3.4, Python 3.5,
```

Comment:
A question not related to this PR. What if we say artman doesn't support python 2.7?

Reply:
Sadly, Python 2.7 is still system Python for Goobuntu and gLinux, as well as many other distributions. Dropping support for 2.7 would increase the barrier to usage pretty dramatically.

Reply:
Ack. Makes sense.